package.json

```diff
@@ -124,7 +124,7 @@
     "start": "stripes serve",
     "test:unit": "jest --ci --coverage",
     "test:e2e": "stripes test karma --karma.singleRun --karma.browsers ChromeDocker --karma.reporters mocha junit --coverage",
-    "test": "yarn run test:e2e && yarn run test:unit"
+    "test": "yarn run test:e2e"
```

Comment:
I don't think we need to delete running of unit tests

Reply:
@AndreiShumski2 I mean it should be `"test": "yarn run test:unit && yarn run test:e2e"`